spec/models/post_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Just as you have added tests for the Post model, kindly add tests for the Like model.

- [ ] Kindly add tests for the Comment model.